v.1.2.4 fails to build from source on FreeBSD

**Describe the bug**

On FreeBSD vault is compiled from source, using build instructions from hashicorp repo.
Currently this fails during `static-assets` step due to missing `pkg/web_ui` dir

```
$ git clean -fdx && git reset --hard
HEAD is now at 360b3a32c release: stage v1.2.4

$ go version
go version go1.13.4 freebsd/amd64

$ env GOPATH=${PWD} PATH=${PWD}/bin:${PATH} gmake bootstrap static-assets static-dist
Installing/Updating golang.org/x/tools/cmd/goimports
Installing/Updating github.com/elazarl/go-bindata-assetfs/...
Installing/Updating github.com/hashicorp/go-bindata/...
Installing/Updating github.com/mitchellh/gox
Installing/Updating github.com/kardianos/govendor
Installing/Updating github.com/client9/misspell/cmd/misspell
Installing/Updating github.com/golangci/golangci-lint/cmd/golangci-lint
--> Generating static assets
bindata: Failed to stat input path 'pkg/web_ui': lstat pkg/web_ui: no such file or directory
Error: go-bindata:  exit status 1
gmake: *** [Makefile:141: static-assets] Error 1

```

**To Reproduce**

Steps to reproduce the behavior:

```
$ sh
$ cd /tmp/
$ curl -#L https://github.com/hashicorp/vault/archive/v1.2.4.tar.gz | tar xzf -
$ cd vault-1.2.4
$ env GOPATH=${PWD} PATH=${PWD}/bin:${PATH} gmake bootstrap static-assets static-dist
```

* Server Operating System/Architecture:

- FreeBSD 12.1R with Go 1.13.4 compiler

**Additional context**

This presumably is broken for anybody building from git or source tarball at present - see https://github.com/hashicorp/vault/issues/7701 for a similar but not identical issue.

Hi,
I can confirm this affects arch linux as well withInstalling/Updating golang.org/x/tools/cmd/goimports
# golang.org/x/tools/cmd/goimports
loadinternal: cannot find runtime/cgo
Installing/Updating github.com/elazarl/go-bindata-assetfs/...
# github.com/elazarl/go-bindata-assetfs/go-bindata-assetfs
loadinternal: cannot find runtime/cgo
Installing/Updating github.com/hashicorp/go-bindata/...
# github.com/hashicorp/go-bindata/go-bindata
loadinternal: cannot find runtime/cgo
Installing/Updating github.com/mitchellh/gox
# github.com/mitchellh/gox
loadinternal: cannot find runtime/cgo
Installing/Updating github.com/kardianos/govendor
Installing/Updating github.com/client9/misspell/cmd/misspell
Installing/Updating github.com/golangci/golangci-lint/cmd/golangci-lint
--> Generating static assets
bindata: Failed to stat input path 'pkg/web_ui': lstat pkg/web_ui: no such file or directory
Error: go-bindata:  exit status 1
make: *** [Makefile:141: static-assets] Error 1
 version 1.3.0:
```

--- a/Makefile
+++ b/Makefile
@@ -17,7 +17,7 @@ EXTERNAL_TOOLS=\
 	github.com/kardianos/govendor \
 	github.com/client9/misspell/cmd/misspell \
 	github.com/golangci/golangci-lint/cmd/golangci-lint
-GOFMT_FILES?=$$(find . -name '*.go' | grep -v pb.go | grep -v vendor)
+GOFMT_FILES?=$$(find . -name '*.go' | grep -v pb.go | grep -v vendor | grep -v testdata)
 
 GO_VERSION_MIN=1.12.7
 CGO_ENABLED?=0
@@ -136,7 +136,10 @@ bootstrap:
 update-plugins:
 	grep vault-plugin- vendor/vendor.json | cut -d '"' -f 4 | xargs govendor fetch
 
-static-assets:
+static-assets-dir:
+	@mkdir -p ./pkg/web_ui
+
+static-assets: static-assets-dir
 	@echo "--> Generating static assets"
 	@go-bindata-assetfs -o bindata_assetfs.go -pkg http -prefix pkg -modtime 1480000000 -tags ui ./pkg/web_ui/...
 	@mv bindata_assetfs.go http

